mmdet3d/core/bbox/coders/km3d_decoder.py

```diff
@@ -0,0 +1,498 @@
+import numpy as np
+import torch
+from torch.nn import functional as F
+
+
+def _nms(heat, kernel=3):
+    """Process heat with Non Maximum Suppression.
+git
+    Args:
+        heat (torch.Tensor): Predictions from model in shape (batch_size *
+            channels * height * width)
+        kernel (int): kernel size for max pooling
+
+    Returns:
+        torch.Tensor: batch_size * channels * height * width
+    """
+    pad = (kernel - 1) // 2
+    hmax = F.max_pool2d(heat, (kernel, kernel), stride=1, padding=pad)
+    keep = (hmax == heat).float()
+    return heat * keep
+
+
+def _left_aggregate(heat):
+    """Process heat with left aggregate.
```

Comment:
Describe the 'left_aggregate' more specifically or use another clearer function name